src/main/java/com/example/school/model/Student.java

```diff
@@ -0,0 +1,7 @@
+package com.example.school.model;
+
+public class Student {
```

Comment:
It would be better add modifiers to all fields in all models. And using `lombok` add getter and setter and constructor.

Reply:
Also in class `Main` create a students.(Number of students should be like we discussed in the task)

Reply:
Done